No module named 'asyncmy.connection' on python 3.11.3

When running tests on python 3.11.3 I got a stack trace:

```
Traceback:
../../.pyenv/versions/3.11.3/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_connection_options.py:14: in <module>
    from databases.backends.asyncmy import AsyncMyBackend
databases/backends/asyncmy.py:6: in <module>
    import asyncmy
../../.pyenv/versions/3.11.3/lib/python3.11/site-packages/asyncmy/__init__.py:1: in <module>
    from .connection import Connection, connect  # noqa:F401
E   ModuleNotFoundError: No module named 'asyncmy.connection'
```
The issue was discussed in [`asyncmy` issue](https://github.com/long2ice/asyncmy/issues/55) and was fixed in [v0.2.7](https://github.com/long2ice/asyncmy/releases/tag/v0.2.7) version.

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 -e .
 
 # Async database drivers
-asyncmy==0.2.5
+asyncmy==0.2.7
 aiomysql==0.1.1
 aiopg==1.3.4
 aiosqlite==0.17.0

